pkg/services/control/ir/types.proto

```diff
@@ -18,12 +18,15 @@ enum HealthStatus {
     // Undefined status, default value.
     HEALTH_STATUS_UNDEFINED = 0;
 
+    // Create IR server
+    CREATE_SERVER = 1;
+
     // IR application is starting.
-    STARTING = 1;
+    STARTING = 2;
 
     // IR application is started and serves all services.
-    READY = 2;
+    READY = 3;
```

Comment:
i think you have broken the older version of cli that expect `2` as READY: https://github.com/nspcc-dev/neofs-node/blob/4a1bc798d2c5cbfd366981528745132c20b09f42/cmd/neofs-cli/modules/control/healthcheck.go#L70-L72

it is internal API but still i think it is not worth it, @roman-khimov

Reply:
also, FIY, if we do such things we usually add something to our CHANGELOG to prepare our users for their catastrophic update, e.g.: https://github.com/nspcc-dev/neofs-node/blob/4a1bc798d2c5cbfd366981528745132c20b09f42/CHANGELOG.md?plain=1#L154-L170

Reply:
Compatibility better be kept, it doesn't cost a lot.

Reply:
Added new statuses after the existing ones, therefore, it no longer breaks.